### ISSUE TITLE
Infrastructure: bump igluctl to 0.8.0

### ISSUE THREAD

**Issue by chuwy** [snowplow/iglu-central#1128]
There are some important API changes in 0.8.0: https://github.com/snowplow-incubator/igluctl/pull/81.

Mostly we've made it more restrictive in the way that it fails `static s3cp` if there are any incompatibilities and `s3cp` now [generates schema list files](https://github.com/snowplow-incubator/igluctl/issues/63) by default. However, it's also worth pointing out that Iglu Central CI/CD relies on `static s3cp` command [only for dev registries](https://github.com/snowplow/iglu-central/blob/ffdd27da2590b8f54113f8cae996db16a1a4622b/.github/scripts/dev_mirror_sync.bash#L27) and main sync [relies on AWS CLI](https://github.com/snowplow/iglu-central/blob/7b444acca8bd2ae75e6318b3ee456fb057fd3d58/.github/scripts/push.bash#L32).

In other words, I think that if the changes affect only `s3cp`, the bump should be safe.

**Comment by istreeter** [comment]
I think it would be great to also start using `static s3cp` for the main sync, so that the main iglu central registry has the schema lists.  It would remove a bar to entry for new snowplow users, because they could get started with iglu-central schemas without needing to deploy their own iglu server for loaders to work.

However, we should be careful about how we make that change.  Snowplow pipelines that have iglu-central in the resolver.json could suddenly start resolving schema lists differently.  I do not expect that to break a pipeline.... but it is a significant change in pipeline behaviour, and therefore would be irresponsible to make that change without giving notice to open source users.

I understand there can be ambiguity in statically generated schema lists generated by igluctl - e.g. which should come first in the list out of `1-0-1` or `1-0-0`.  Iglu Central only has *one* schema with an ambiguity, and it is `com.snowplowanalytics.snowplow.storage/postgresql_config`, so I think this is low risk.
